src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -977,6 +996,14 @@ protected boolean isEnabled(String locator) {
         return true;
     }
 
+    protected boolean isFocused(String locator) {
+        List<WebElement> elements = elementFind(locator, true, true);
+        WebElement element = elements.get(0);
```

Comment:
What if there are no elements with a specified locator?

Reply:
Ah yes, good catch... I'll ad an if statement to catch that problem. I took this piece of code from one of the isEnabled though, so that same problem would be in there as well probably. I'll check that as well, see if the same problem exists elsewhere

Reply:
just did a little checking. This is handled in the elementFind by setting the required parameter to true. elementFind then does:

```
if (required && elements.size() == 0) {
            throw new SeleniumLibraryNonFatalException(
                    String.format("Element locator '%s' did not match any elements.", locator));
        }
```

so there is no need to do that again in my isFocused method.